src/client/steam/interfaces/apps.hpp

```diff
@@ -7,6 +7,29 @@ namespace steam
 	public:
 		~apps() = default;
 
+		apps()
+		{
+			if (std::filesystem::exists(LanguageFileName))
+			{
+#if DEBUG
+				printf("[Language] Custom language file has been found.\n");
+#endif
+
+				std::ifstream file;
+				file.open(LanguageFileName);
+				std::string line;
+				while(std::getline(file, line))
+					Language = line.c_str();
```

Comment:
Just use `utils::io::file_exists` and `utils::io::read_file`.
Also `std::string line` is a local variable which will be deleted after the execution of `apps()` so you should make `const char* Language` a char array and copy the result of `utils::io::read_file` to it